contracts/SubdomainRegistrar.sol

```diff
@@ -18,7 +19,7 @@ import "./RegistrarInterface.sol";
  * ownership in the ENS registry to this contract. Ownership in the contract
  * may be transferred using `transfer`, and a domain may be unlisted for sale
  * using `unlistDomain`. There is (deliberately) no way to recover ownership
- * in ENS once the name is transferred to this registrar.
+ * in ENS once the name is transferred to this hashRegistrar.
```

Comment:
It's not a `hashRegistrar`

Reply:
Also, "is the deed held by a custodian contract" (comment below) will now be checked, since this is the custodian contract.